docs/General/manageDevice.md

```diff
@@ -16,11 +16,15 @@
     [![mps](../assets/images/MPS_ManageDevice.png)](../assets/images/MPS_ManageDevice.png)
 
     !!! note
-        If activated into Client Control Mode(CCM), the KVM and SOL features will not function in the current release. To use KVM/SOL, follow the [ACM Activation Tutorial](createProfileACM.md) to see how to configure a device into Admin Control Mode.
+        If activated into Client Control Mode(CCM), the KVM and SOL features uses user consent code. To use KVM/SOL, follow the [ACM Activation Tutorial](createProfileACM.md) to avoid user consent and how to configure a device into Admin Control Mode.
```

Comment:
If activated into Client Control Mode(CCM), the KVM and SOL features will require a user consent code displayed on the device. To use KVM/SOL, follow the [ACM Activation Tutorial](createProfileACM.md) to avoid user consent and how to configure a device into Admin Control Mode.

Reply:
done